app/controllers/track_controller.rb

```diff
@@ -220,10 +220,8 @@ def delete_all_type
     track_type = params[:track_type]
 
     flash[:notice] = _("You will no longer be emailed updates for those alerts")
-    for track_thing in TrackThing.find(:all, :conditions => [ "track_type = ? and tracking_user_id = ?", track_type, user_id ])
-      track_thing.destroy
-    end
-
+    TrackThing.where(:track_type => track_type, :tracking_user_id => user_id) \
+      .destroy_all
```

Comment:
I think the trailing `.` chaining style prevents the need for a trailing `\`, which is nicer:

```ruby
TrackThing.
  where(:track_type => track_type, :tracking_user_id => user_id).
    destroy_all
```

I have no idea why a trailing `\` is required though. It should [work without](https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains).